### ISSUE TITLE
GRID dataset

### ISSUE THREAD

**Issue by isxuwl02** [mpc001/Visual_Speech_Recognition_for_Multiple_Languages#21]
Could you please share the original GRID data set? There are some missing items online.

**Comment by Chhaya-Trootech** [comment]
yes, i also facing issue, here rnnlm and rnnlm_conf is empty, and it is not able to work accurately. i mean WER is higher than 1.2. 

 [input]
modality=video
v_fps=25

[model]
v_fps=25
model_path=benchmarks/GRID/models/GRID_V_WER1.2/model.pth
model_conf=benchmarks/GRID/models/GRID_V_WER1.2/model.json
rnnlm=
rnnlm_conf=

[decode]
beam_size=20
penalty=0.
maxlenratio=0.0
minlenratio=0.0
ctc_weight=0.1
lm_weight=0.
